packages/core/src/oidc/grants/index.ts

```diff
@@ -24,4 +25,15 @@ export const registerGrants = (oidc: Provider, envSet: EnvSet, queries: Queries)
     refreshToken.buildHandler(envSet, queries),
     ...parameterConfig
   );
+
+  // Token exchange grant
+  const tokenExchangeParameterConfig: [parameters: string[], duplicates: string[]] =
```

Comment:
what is the difference between `tokenExchangeParameterConfig` and `parameterConfig`?

Reply:
you can rebase and use `getParameterConfig` for this

Reply:
> what is the difference between `tokenExchangeParameterConfig` and `parameterConfig`?

separate from the parameterConfig above - this can be refactored using `getParameterConfig` in the master branch

Reply:
updated